src/lib/bans.ts

```diff
@@ -20,7 +83,7 @@ export function isBptfBanned(steamID: SteamID | string, bptfApiKey: string, user
             {
                 url: 'https://backpack.tf/api/users/info/v1',
                 headers: {
-                    'User-Agent': 'TF2Autobot@' + process.env.BOT_VERSION,
```

Comment:
Is there a rationale for not identifying the specific version of the software in the user agent?

Just sending the name is sufficient to meet the [RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-5.5.3). Providing a version identifier as well could be beneficial because that gives web site operators more discretion as to how they deal with TF2Autobot agents in general. For example if an older version was malfunctioning the site could choose to deal with that specific version versus perhaps blocking the whole agent ID.

Reply:
It was removed because @Nicklason said it's good, but perhaps it's better to keep it.

Reply:
I guess I'll revert the changes made in #888.

Thank you